pkg/auth/oidc_connect.go

```diff
@@ -323,31 +314,33 @@ func parseURL(req *Request) *url.URL {
 	return u
 }
 
-//  formatRequest safe to ignore only use during debugging
-func formatRequest(r *Request) string {
-
-	// Create return string
-	var request []string
-	// Add the request string
-
-	u := fmt.Sprintf("%v %v %v %v", r.Request.Method, r.Request.URL, r.Request.RequestURI, r.Request.Proto)
-	request = append(request, u)
-	// Add the host
-	request = append(request, fmt.Sprintf("Host: %v", r.Request.Host))
-	// Loop through headers
-	for name, headers := range r.Request.Header {
-		//name = strings.ToLower(name)
-		for _, h := range headers {
-			request = append(request, fmt.Sprintf("%v: %v", name, h))
-		}
-	}
-
-	// If this is a POST, add post data
-	if r.Request.Method == "POST" {
-		r.Request.ParseForm()
-		request = append(request, "\n")
-		request = append(request, r.Request.Form.Encode())
-	}
-	// Return the request as a string
-	return strings.Join(request, "\n")
-}
+//// formatRequest safe to ignore only use during debugging
```

Comment:
could this be usable for a debug level log? or removable?

Reply:
It's not used anywhere, so for now could just be removed.